tech/languages/ocaml/dune.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Dune
+subsection: ocaml
+order: 2
+---
+
+# Dune
+
+Dune is a build system for OCaml. It provides a comprehensive and uniform interface for the building, testing, and installation of OCaml projects, making it incredibly easy for developers to manage dependencies between packages. Its primary focus is to offer simplicity, speed, and ease of use, making it a suitable solution for even the most complex and extensive OCaml projects. With Dune, you can enjoy the benefits of incremental builds, which result in faster build times by recompiling only the necessary files. Moreover, Dune makes it convenient to generate optimized native code and perform static analysis, contributing to the development of high-performance and reliable OCaml applications.
+
+## Install
+
+You can install Dune with OPAM that we installed in the enviroment setup. For do it just run:
+
+```console
+$ opam install dune
+```
+
+## Starting a new Project
+
+1. Create a new directory for your project.
```

Comment:
numbering in this whole section turns into all 1s.  I think the numbering has to be in succession if you want it numbered just like here: which renders correctly the 1 and 2 
```
## Building the project.
1...
2...
```

Reply:
Fixed https://github.com/developer-portal/content/pull/473/commits/7e02639e7f9c88ea9411215e7265a21d3926080a